OpenTerm/Controller/ViewController.swift

```diff
@@ -339,6 +357,261 @@ extension ViewController: UIDocumentPickerDelegate {
 
 }
 
+extension ViewController: BookmarkViewControllerDelegate {
+    
+    /// The file name of the start directory bookmark. It is stored as a static
+    /// constant so we can access it when saving or loading the bookmark and
+    /// such that the settings view controller can access is as well.
+    static let bookmarkDirectory = ".bookmarks"
+    
+    
+    /// Gets the URLs of the bookmarks that were previously saved.
+    ///
+    /// - Returns: The URLs of the saved bookmarks. If something fails, the returned array will be empty.
+    func savedBookmarkURLs() -> [URL] {
```

Comment:
Probably best to keep all of this data loading logic in a separate helper class, since we don't want `ViewController` to get too big. See the `HistoryManager` class as an example.

Reply:
Yep, that's a good idea. I will have a look.